frontend/test/users/users.redux.spec.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ import {
+	INITIAL_STATE,
+	usersReducer,
+	UsersActions,
+	IUser
+} from '@/v5/store/users/users.redux';
+import { selectUser, selectUsersByTeamspace } from '@/v5/store/users/users.selectors';
+
+
+describe('Users: redux', () => {
+	const teamspace = 'teamspaceA';
+
+	const bob = {
+		user: 'bob',
+		firstName: 'Bob',
+		lastName: 'User',
+		company: 'Company A',
+		job: '',
+		email: 'bob@companya.com',
+		hasAvatar: false,
+		avatarUrl: '',
+	}
+
+	const alice = {
+		user: 'alice',
+		firstName: 'Alice',
+		lastName: 'Member',
+		company: 'Company B',
```

Comment:
Alice has bob's email and belongs to company B instead of company A. I know this is a test file so it is not a big deal, but I think it would still be better to have Alice and Bob object to be more precise.